src/Cloud/weglide/WeGlideSettings.hpp

```diff
@@ -55,8 +65,7 @@ struct WeGlideSettings {
   static constexpr char gliderlist_uri[] = "https://raw.githubusercontent.com/"
     "weglide/GliderList/master/gliderlist.csv";
 
-  uint32_t pilot_id;
-  BrokenDate pilot_birthdate;
+  struct WeGlide::Pilot pilot;
```

Comment:
Redundant `struct` keyword. Not needed in C++.

Reply:
But I need this foreward declaration for the next function declaration? Otherwise I had to include the whole `WeGlideSettings.hpp`, but this I don't like?
I only see the pointless 'WeGlide::' because this declaration is inside the namespace!

Reply:
Please, excuse me: This isn't the forward declaration - this is the variable definition - and here the strucct keyword is wrong, of course!